src/content/en/updates/2017/12/chrome-64-media-updates.md

```diff
@@ -0,0 +1,349 @@
+  videoCapabilities: [{
+    contentType: 'video/webm; codecs="vp9"',
+    robustness: 'SW_SECURE_DECODE' // Widevine L3
+  }]
+}];
+
+navigator.requestMediaKeySystemAccess('com.widevine.alpha', config)
+.then(access => {
+  // User will be able to watch encrypted content while being offline when
+  // license is stored locally on device and loaded later.
+})
+.catch(error => {
+  // Persistent licenses are not supported on this platform yet.
+});
+```
+
+You can try persistent licenses yourself by checking out the [Sample Media PWA]
+and following these steps:
+
+1. Go to [https://biograf-155113.appspot.com/ttt/episode-2/]
+2. Click "Make available offline" and wait for the video to be downloaded.
+3. Turn off internet connection.
+4. Click the "Play" button and enjoy the video!
+
+## Media preload defaults to "metadata" {: #media-preload-defaults-metadata }
+
+Matching other browsers implementations, Chrome desktop now sets the default
+preload value for `<video>` and `<audio>` elements to "metadata" in order to
+reduce bandwidth and resource usage. This new behaviour only applies to cases
+where no preload value is set. Note that the preload attribute's hint is
+discarded when a `MediaSource` is attached to the media element as the web site
+handles its own preload.
+
+In other words, `<video>` preload value is now "metadata" while `<video
+preload="auto">` preload value stays "auto". Give a try to the [official
+sample](https://googlechrome.github.io/samples/media/preload-metadata).
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/5CDvJkdxyQ8) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5682169347309568) &#124;
+[Chromium Bug](https://crbug.com/310450)
+
+## Unsupported playbackRate raises an exception {: #unsupported-playbackRate-raises-exception }
+
+Following an [HTML specification change], when media elements' playbackRate
+is set to a value not supported by Chrome (eg. negative value), a
+"NotSupportedError" DOMException is thrown in Chrome 63.
+
+```js
+const audio = document.querySelector('audio');
+try {
+  audio.playbackRate = -1;
+} catch(error) {
+  console.log(error.message); // Failed to set the playbackRate property
+}
+```
+
+For info, Chrome’s current implementation raises this exception when new
+playbackRate is either negative, less than 0.0625, or more than 16. Give a try
+to the [official
+sample](https://googlechrome.github.io/samples/media/playback-rate-exception)
+to see this in action.
+
+[Intent to Ship](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/a0tguvcZZyk) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5750156230131712) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=747082)
+
+## Background video track optimizations {: #background-video-track-optimizations }
+
+The chrome team is always trying to find new ways to improve battery life and
+Chrome 63 was no exception.
```

Comment:
Why are we suddenly in Chrome 63? If you're mixing updates from two versions, I suggest that you don't. Split them into two articles so they can be properly tagged.

Reply:
There are two minor items in Chrome 63 and instead of having a ridiculously short article, we agreed to append them in this post.

Shall I rename this article to "Audio/Video Updates in Chrome 63-64"? Or just leave it as it as mentioning Chrome 63 is enough for these small nits?

Reply:
Yes. By the way. Who's 'we'. I don't have a problem with a short article, if it's appropriate.

Reply:
'we' was the Chrome media team who've reviewed the draft. I'll rename it now.